apps/stats/management/commands/__init__.py

```diff
@@ -92,3 +101,24 @@ def get_date_from_file(filepath, sep):
             return line.split(sep)[0]
         except IndexError:
             return None
+
+
+def stats_from_model(model):
+    """Return the stats from the model ready to write to a file."""
+    data = model_to_dict(model)
+    del data['date']  # Already in the filename.
+    del data['addon']  # Already in the file's folder name.
+    del data['id']  # No need for the model's ID at all (eg: UpdateCount).
+    return json.dumps(data)
```

Comment:
oh, I see, I can add the date and addon back, but it really should already be in the file/folder names. The `addon` here is the addon id, `id` is the id of the model holding the stats (which isn't relevant at all here).

It feels like a lot of redundancy, what do you think?